<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Second</class>
 <widget class="QWidget" name="Second">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableQ">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>221</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Text</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Score</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="answerButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>88</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Score</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditScore"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEditAnswer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
